snappy.cc

```diff
@@ -1245,7 +1245,7 @@ class SnappyScatteredWriter {
   inline bool AppendFromSelf(size_t offset, size_t len) {
     // See SnappyArrayWriter::AppendFromSelf for an explanation of
     // the "offset - 1u" trick.
-    if (offset - 1u < op_ptr_ - op_base_) {
+    if (offset - 1u < size_t(op_ptr_ - op_base_)) {
```

Comment:
Can you switch casts to [C++-style casts ](https://google.github.io/styleguide/cppguide.html#Casting)?